src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -359,4 +370,51 @@ private function getContainerEnvVars(ContainerBuilder $container): array
 
         return array_values($envs);
     }
+
+    protected function getUsagesForDefinition(Definition $definition)
```

Comment:
```suggestion
    protected function getUsagesForDefinition(Definition $definition): array
```

Try to set return type declarations.

Reply:
Fixed!